src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.Net.Sdk.WorkloadManifestReader
+{
+    public class SdkDirectoryWorkloadManifestProvider : IWorkloadManifestProvider
+    {
+        readonly string sdkRootPath;
+        readonly string sdkVersionBand;
+
+        public SdkDirectoryWorkloadManifestProvider(string sdkRootPath, string sdkVersionBand)
```

Comment:
string sdkRootPath, string sdkVersionBand should be parameter for GetManifests. You cannot dependency inject with sdkRootPath and sdkVersionBand required in constructor